gradle/docker.gradle.kts

```diff
@@ -33,106 +34,105 @@ logger.lifecycle(">> dockerContainerName: $dockerContainerName")
+        doLast {
+            if (errorOutput.toString().contains("Error: No such network")) {
+                //we are happy, since this network is down we can proceed
+            } else {
+                Thread.sleep(15 * 1000) // let's wait for better times
+            }
         }
+        mustRunAfter("build")
     }
-    mustRunAfter("build")
-}
-
-tasks.register<Exec>("deployLocal") {
-    environment = mapOf("COGBOARD_VERSION" to version)
-    group = "swarm"
-    commandLine = listOf("docker", "stack", "deploy", "-c", "${project.name}-local-compose.yml", "${project.name}-local")
-    dependsOn("initSwarm", "buildImage", "awaitLocalStackUndeployed")
-    mustRunAfter("undeployLocal")
-}
 
-tasks.register<Exec>("undeployLocal") {
-    group = "swarm"
-    commandLine = listOf("docker", "stack", "rm", "${project.name}-local")
-    dependsOn("initSwarm")
-}
+    register<Exec>("deployLocal") {
+        environment = mapOf("COGBOARD_VERSION" to version)
+        group = "swarm"
+        commandLine = listOf("docker", "stack", "deploy", "-c", "${project.name}-local-compose.yml", "${project.name}-local")
+        dependsOn("initSwarm", "buildImage", "awaitLocalStackUndeployed")
+        mustRunAfter("undeployLocal")
+    }
 
-tasks.register("redeployLocal") {
-    group = "swarm"
-    dependsOn("undeployLocal", "deployLocal")
-}
+    register<Exec>("undeployLocal") {
+        group = "swarm"
+        commandLine = listOf("docker", "stack", "rm", "${project.name}-local")
+        dependsOn("initSwarm")
+    }
 
-tasks.register<Exec>("functionalTests") {
-    group = "docker-functional-tests"
-    commandLine = listOf("docker", "run", "-v","$functionalTestsPath:/e2e","-w","/e2e","--network=$network", "cypress/included:3.8.0", "--browser", "chrome", "--config-file", "$cypressConfigPath")
+    register("redeployLocal") {
+        group = "swarm"
+        dependsOn("undeployLocal", "deployLocal")
+    }
 
-    dependsOn("redeployLocal", "copyCypressContent" )
-    doFirst {
-        logger.lifecycle("Running functional tests from $functionalTestsPath with base network: $network")
+    register<Exec>("functionalTests") {
+        group = "Docker Functional Tests"
+        commandLine = listOf(
+                "docker", "run",
+                "-v", "$functionalTestsPath:/e2e",
+                "-w", "/e2e",
+                "--network=$network",
+                "cypress/included:3.8.0",
+                "--browser", "chrome",
+                "--config-file", cypressConfigPath
+        )
+
+        dependsOn("redeployLocal", "copyCypressContent")
+        doFirst {
+            logger.lifecycle("Running functional tests from $functionalTestsPath with base network: $network")
+        }
     }
-}
 
-tasks.register<Exec>("copyCypressContent") {
-    group = "docker-functional-tests"
-    commandLine = listOf("sh", "scripts/copyCypressContent.sh", "$cypressContentPath", "$mountDir")
-    mustRunAfter("redeployLocal")
```

Comment:
We have better approach for copying files - lets use the old way without unnecessary script that will not work when you don't have shell `sh` installed on your system.